packages/webapp/webapp_server.js

```diff
@@ -266,30 +264,24 @@ var getBoilerplate = function (request, arch) {
     }
     return memoizedBoilerplate[memHash];
   }
-  
-  var boilerplateOptions = _.extend({ 
-    htmlAttributes: htmlAttributes 
+
+  var boilerplateOptions = _.extend({
+    htmlAttributes: htmlAttributes
   }, _.pick(request, 'dynamicHead', 'dynamicBody'));
```

Comment:
As long as we're touching this code, I'll just note that this could be
```js
var boilerplateOptions = {
  htmlAttributes,
  ..._.pick(request, "dynamicHead", "dynamicBody")
}
```